docs/content/content/content-styles.mdx

```diff
@@ -5,6 +5,126 @@ complete: no
 order: 4
 ---
 
+## Abbreviations
+
+An abbreviation is a shortened form of a word. It contains the first letter or more of a word generally. They don’t include a word’s last letter. 
```

Comment:
"They don’t include a word’s last letter." WAT?!

Reply:
Weird, but it's in the style guide. For your reference: https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/shortened-words-and-phrases/abbreviations